lib/checks/aria/invalidrole.js

```diff
@@ -1,3 +1,7 @@
-return !axe.commons.aria.isValidRole(node.getAttribute('role'), {
-	allowAbstract: true
-});
+return (
+	axe.utils.tokenList(virtualNode.attr('role')).filter(role => {
+		return !axe.commons.aria.isValidRole(role, {
+			allowAbstract: true
+		});
+	}).length > 0
```

Comment:
Can you put the output of this into data, and list which roles are invalid? Same thing with abstract role. I know that's not strictly part of this PR, but it'd make the messaging about this a bunch better.